### DISCUSSION TITLE
2022-05-06 Explorer Team Updates

### DISCUSSION THREAD

**Discussion by frol** [near/community#268] [Pagoda Platform]
Pshhh, Explorer update is here! 📻 

1. Proof-of-concept FT assets display was implemented (no proper design is there yet).

* Example of FT assets list on an account [`/accounts/vlad.near`](https://frontend-pr-969-kgmz.onrender.com/accounts/vlad.near) ([PR #969](https://github.com/near/near-explorer/pull/969))
* List of all FT tokens (that support FT Events): [`/contracts/fungible-tokens`](https://frontend-pr-957-d14z.onrender.com/contracts/fungible-tokens) ([PR #957](https://github.com/near/near-explorer/pull/957))

2. We have been working on a **second iteration** of Account Activity List and Transaction Execution Plan displays and we arrived to a new UI layout that we will let you, folks check and provide the feedback about with the preview links once we implement it.

    <img width="857" alt="image" src="https://user-images.githubusercontent.com/304265/167419537-590524ac-0699-4a58-8a18-ec19a8cabd58.png">

    Compare it with how the things look right now: https://explorer.near.org/transactions/Bkczfr7YkkY5hZHmUDDc8M22VF3Gej9wEXx1pkuZmeq5 (it is too long to even post the screenshot here)

3. Along our way we pushed [Data Platform team to work on the dedicated account activity table](https://github.com/near/community/discussions/265) on the Indexer for Explorer side, and Data Platform team is currently working on that task.

---

If you have any questions, feel free to drop us a message here!

**Comment by idea404** [comment]
@frol considering you guys will be moving to SingleStore DB and we have the NEAR lake indexer out there, will the public postgresDB be deprecated?

**Reply by frol** [reply]
@idea404 I feel we will only deprecate it when it is clear that all other solutions are strongly better than the existing one. So far we still see benefits of having Postgres. We are, however also taking a note of AlloyDB and Aurora PostgreSQL (Postgres-compatible solutions)
